tests/common/gcp_api/securitycenter_test.py

```diff
@@ -38,7 +38,7 @@ def setUpClass(cls, mock_google_credential):
         """Set up."""
         fake_global_configs = {
             'securitycenter': {'max_calls': 1, 'period': 1.1}}
-        cls.securitycenter_beta_api_client = securitycenter.SecurityCenterClient(version='v1beta1')
+        cls.securitycenter = securitycenter.SecurityCenterClient(version='v1')
```

Comment:
This would be better as `cls.securitycenter_client`, to match what is being instantiated.

Reply:
Done